raiden/api/python.py

```diff
@@ -1318,5 +1324,13 @@ def withdraw_from_udc(self, amount: TokenAmount) -> TransactionHash:
             amount=amount, given_block_identifier=confirmed_block_identifier
         )
 
+    def get_new_notifications(self):
+        notifications = self.raiden.notifications
+        self.raiden.notifications = defaultdict(lambda x: None)
+        return notifications.items()
+
+    def add_notification(self, notification: Notification) -> None:
+        self.raiden.notifications[notification["id"]] = notification
```

Comment:
I'm not sure if this is the right place for `add_notification`. Since we don't plan to add notifications from outside Raiden, it might be better to remove it from the external API.

Reply:
I moved it to RaidenService now ✅